packages/nx/src/project-graph/nx-deps-cache.ts

```diff
@@ -193,6 +196,7 @@ export function createProjectFileMapCache(
     nxJsonPlugins,
     pluginsConfig: nxJson?.pluginsConfig,
     fileMap,
+    externalNodes: externalNodesData,
```

Comment:
I don't think we should write the entire `externalNodesData`.. can we write just the hash?

It'll save us disk space used by the file.. and it'll save us computing the hash again.

Reply:
Yeah, with the hash, it makes sense. With the output of `JSON.stringify` I had, less so.

If we're interested in reducing disk space used by the file, we should also consider writing an unformatted JSON to disk with no spaces. If anyone wants to manually read it, they can easily format it anyway.